velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -51,6 +50,50 @@ class PrefixSortEncoder {
     }
   }
 
+  /// Encode String type.
+  /// The string prefix is formatted as 'null byte + string content + padding
+  /// zeros'. If `!ascending_`, the bits for both the content and padding zeros
+  /// need to be inverted.
+  FOLLY_ALWAYS_INLINE void encode(
+      const std::optional<StringView>& value,
+      char* dest,
+      uint32_t encodeSize) const {
```

Comment:
Can we pass an optional encodedSize so we only have one encode API? thanks!

Reply:
Updated, thanks.